account_due_list/README.rst

```diff
@@ -1,16 +1,15 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-Account due list
-================
+=====================
+Payments and due list
```

Comment:
This is actually not changed between 9.0 and 10.0
The only changes over 9.0 are https://github.com/OCA/account-payment/pull/129/commits/350dcaf34d748cdb04be2790b57a5e5571b123e6
You can also see them with
```
git diff origin/9.0..eLBati/10.0-mig-account_due_list  account_due_list
```